R/dist.R

```diff
@@ -192,19 +193,21 @@ dist_skel <- function(n, dist = FALSE, cum = TRUE, model,
 #' )
 #'
 #' # integer adjusted lognormal model
-#' dist_fit(rlnorm(1:100, log(5), 0.2),
-#'   samples = 1000, dist = "lognormal",
-#'   cores = ifelse(interactive(), 4, 1), verbose = TRUE
-#' )
+# dist_fit(rlnorm(1:100, log(5), 0.2),
+#   samples = 1000, dist = "lognormal",
+#   cores = ifelse(interactive(), 4, 1), verbose = TRUE
+# )
 #' }
-dist_fit <- function(values = NULL, samples = NULL, cores = 1,
+dist_fit <- function(values = NULL, samples = 1000, cores = 1,
```

Comment:
just need to redoc to get rid of the warning here.

Reply:
I've reverted this change. I'll submit it as a separate PR on fixing defaults.